addons/featured-lists/addon.json

```
@@ -0,0 +1,131 @@
+{
+  "name": "Featured project lists",
+  "description": "Change order of lists that are displayed on the front page.",
```

Possible description:
```suggestion
  "description": "Changes the order of rows on the front page or hides them.",
```